### PR TITLE
Changed glyphicon-info-sign to af-icons-info

### PR DESCRIPTION
**Pull request by Ravaelles**

### CODE REVIEW

**Comment by Ravaelles** [comment]
[(4) Judge assignment on panels](https://trello.com/c/WQkOaB0B/3414-4-judge-assignment-on-panels)

File: src/Multiselect.vue [hunk 1]
```diff
@@ -159,12 +161,20 @@ export default {
     },
     infoIconClass: {
       type: String,
-      default: 'glyphicon glyphicon-info-sign'
+      default: 'af-icons af-icons-info'
```
Comment (antennaio) [inline src/Multiselect.vue:164]: I asked Kirk once about it and he was against making any references to AF in this repository. That's why it uses  Bootstrap's glyphicons, which can be replaced with something else via the prop.

File: src/MultiselectOption.vue [hunk 2]
```diff
@@ -140,4 +140,10 @@ export default {
     overflow: hidden;
     text-overflow: ellipsis;
   }
+
+  .multiselect-option span.af-icons {
```
Comment (antennaio) [inline src/MultiselectOption.vue:144]: Same here, this likely belongs on AF.

File: src/Multiselect.vue [hunk 3]
```diff
@@ -159,12 +161,20 @@ export default {
...
       default: 'glyphicon glyphicon-question-sign'
     },
+    showNothingFound: {
+      type: Boolean,
+      default: false
+    },
+    nothingFoundLabel: {
+      type: String,
+      default: ''
+    },
```
Comment (antennaio) [inline src/Multiselect.vue:177]: Please add the new props to the docs.

I think it's fine to assign a default to the `nothingFoundLabel`, "Nothing found" for example?

**Comment by Ravaelles** [comment]
Hey @antennaio. Thanks for your valuable input!
I changed what you've requested.

**Review by antennaio — APPROVED** [review]
Looks good 👍

**Pull request merged**